includes/class-wc-stripe-payment-method-configurations.php

```diff
@@ -129,13 +129,150 @@ private static function get_payment_method_configuration_from_stripe() {
 			// The API returns data for the corresponding mode of the api keys used, so we'll get either test or live PMCs, but never both.
 			if ( $configuration->parent && ( self::LIVE_MODE_CONFIGURATION_PARENT_ID === $configuration->parent || self::TEST_MODE_CONFIGURATION_PARENT_ID === $configuration->parent ) ) {
```

Comment:
This is unrelated to the changes made in this PR. Right now, we don't really care if the PMC is active or not as long as it is connected to a WC PMC. Is that intentional? What would happen if this PMC is not active?

Reply:
Good question. I _think_ we should limit these checks to active PMCs as a baseline. I can tackle that in a follow-up PR.